docs/ssss.md

```diff
@@ -1,3 +1,106 @@
 # Simperby Super Simple Summary
 
-TODO
```

Comment:
I think it's better to add an index here, though this writing is quite short.

Reply:
I think it's important to keep ssss as short as possible. Let's have indices for long documents only.

Reply:
ok then